app/views/shared/_user.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= link_to user_path(user) do %>
+    <div class="user-container">
+      <div class="img-container">
+        <%= image_tag user.photo %>
+      </div>
+      <div class="user_details">
+        <h2 class="name"><%= user.name %></h2>
+        <p class="nu-of-posts">Number of posts: <%= user.posts_counter %></p>
```

Comment:
- [ ] You wrote posts_counter but according to your schema, it should be post_counter. When I run your program, I get this error message
![image](https://user-images.githubusercontent.com/45460192/222922996-0401b473-5357-4790-b65c-8a59c421cc95.png)

Reply:
I don't get any errors, to be sure I will check it.

Reply:
![posts_counter](https://user-images.githubusercontent.com/60324033/222924164-d13b17ee-6f7a-4bf3-bab7-fff7bde32fb6.PNG)

Reply:
And this is my database
![users_table](https://user-images.githubusercontent.com/60324033/222927271-91f069ba-88bd-476e-8296-bb72ce7ddeab.PNG)

Reply:
Please have a look at my schema also.
![schema](https://user-images.githubusercontent.com/60324033/222927344-6e4b6dca-60bb-4865-8d92-95d375e9bd9b.PNG)